docs/snippets/release_46.md

```diff
@@ -8,4 +8,6 @@ To learn more about all the included changes, see the full release change logs:
 - [[[= product_name_exp =]] [[= version =]]](https://github.com/ibexa/experience/releases/tag/[[= version =]])
 - [[[= product_name_com =]] [[= version =]]](https://github.com/ibexa/commerce/releases/tag/[[= version =]])
 
+[[% if version != 'v4.6.0' %]]
 To update your application, see the [update instructions](../update_and_migration/from_4.6/update_from_4.6.md#[[= version_to_anchor(version) =]]).
```

Comment:
Is there something preventing autolink here?
```suggestion
To update your application, see the [update instructions](update_from_4.6.md#[[= version_to_anchor(version) =]]).
```

Reply:
Yes, it doesn't work and generates a lot of:
```
WARNING -  Doc file 'release_notes/ibexa_dxp_v4.6.md' contains a link 'update_from_4.6.md#v461', but the target 'release_notes/update_from_4.6.md' is not found
           among documentation files.
```

I think autolinkining doesn't work in snippets, somehow the context of "current file" is lost

Reply:
OK. So the relative path has to stay.